media-libs/ctl/files/ctl-1.5.2-Use-GNUInstallDirs-and-fix-PkgConfig-files.patch

```diff
@@ -0,0 +1,371 @@
+From e284bd943f7c72b13c4b2bd508227ba3657ff6ca Mon Sep 17 00:00:00 2001
+From: Jonathan Scruggs <j.scruggs@gmail.com>
+Date: Mon, 25 Sep 2017 17:41:26 +0100
+Subject: [PATCH] CTL: Use GNUInstallDirs and fix PkgConfig files
```

Comment:
Please add a link to the upstream PR if not merged yet

Reply:
Sorted